draft-duke-quic-load-balancers.md

```diff
@@ -318,26 +318,24 @@ formats presented below.
 
 ## QUIC-LB Message Exchange
 
-QUIC-LB load balancers send the encoding parameters to servers using a QUIC-LB
-packet.  They periodically retransmit this packet to each server until that
-server responds with an acknowledgement. Specifics of this retransmission are
-implementation-dependent.
-
-These packet formats are specific to QUICv1 and experimental versions leading up
-to QUICv1. They may require revision for future versions of QUIC.
+QUIC-LB load balancers and servers exchange messages via the QUIC-LBv1 protocol,
+which uses the QUIC invariants with version number 0xF1000000. The QUIC-LB
+load balancers send the encoding parameters to servers and periodically
+retransmit until that server responds with an acknowledgement. Specifics of this
+retransmission are implementation-dependent.
 
 ## QUIC-LB Packet {#quic-lb-packet}
 
-A QUIC-LB packet uses a long header with a type value of 0xFB.  It carries
-configuration information from the load balancer and acknowledgements from the
-servers. They are sent when a load balancer boots up, detects a new server in
-the pool or needs to update the server configuration.
+A QUIC-LB packet uses a long header.  It carries configuration information from
+the load balancer and acknowledgements from the servers.  They are sent when a
+load balancer boots up, detects a new server in the pool or needs to update the
+server configuration.
 
 ~~~~~
 0                   1                   2                   3
 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
 +-+-+-+-+-+-+-+-+
-| Type = 0xfb   |
+|1|C R| Reserved|
```

Comment:
Should we grease the Reserved bits?

Reply:
I don't think we need to worry about it. I don't expect this to get ossified. It won't be on the open internet (generally).

Reply:
It seems low-cost to grease but I don't care enough to make the change.